packages/media/src/getResourceSize.ts

```diff
@@ -19,6 +19,14 @@
  */
 import { FULLBLEED_HEIGHT, PAGE_WIDTH } from '@googleforcreators/units';
```

Comment:
So this `Could not find a declaration file for module '@googleforcreators/units'.` error has been nagging me for a while now and can't figure out why it's not working 🤔

Reply:
Curious, what was the issue?